tests/contrib/test_gce.py

```diff
@@ -13,240 +13,252 @@
 # limitations under the License.
 
 """Unit tests for oauth2client.contrib.gce."""
-
+from datetime import datetime
```

Comment:
[Import only modules, not classes, functions, or other code elements of modules.](https://google.github.io/styleguide/pyguide.html?showone=Imports#Imports)